docs/development/best-practices.md

```diff
@@ -0,0 +1,113 @@
+# Best Practices
+
+## General
+
+- Prefer full function declaration (as opposed to `const func = ():void => {}`) for readability and better stack traces
```

Comment:
```suggestion
- Prefer full function declaration, as opposed to `const func = ():void => {}`, for readability and better stack traces
```

Just using `,` is clearer, and removes the confusion between the text `()` and the code `()`.